manage_breast_screening/participants/models.py

```diff
@@ -158,34 +168,43 @@ def filter_counts_for_clinic(self, clinic):
 
 
 class Appointment(BaseModel):
-    class Status:
-        CONFIRMED = "CONFIRMED"
-        CANCELLED = "CANCELLED"
-        DID_NOT_ATTEND = "DID_NOT_ATTEND"
-        CHECKED_IN = "CHECKED_IN"
-        SCREENED = "SCREENED"
-        PARTIALLY_SCREENED = "PARTIALLY_SCREENED"
-        ATTENDED_NOT_SCREENED = "ATTENDED_NOT_SCREENED"
-
-    STATUS_CHOICES = {
-        Status.CONFIRMED: "Confirmed",
-        Status.CANCELLED: "Cancelled",
-        Status.DID_NOT_ATTEND: "Did not attend",
-        Status.CHECKED_IN: "Checked in",
-        Status.SCREENED: "Screened",
-        Status.PARTIALLY_SCREENED: "Partially screened",
-        Status.ATTENDED_NOT_SCREENED: "Attended not screened",
-    }
-
     objects = AppointmentQuerySet.as_manager()
 
     screening_episode = models.ForeignKey(ScreeningEpisode, on_delete=models.PROTECT)
     clinic_slot = models.ForeignKey(
         "clinics.ClinicSlot",
         on_delete=models.PROTECT,
     )
-    status = models.CharField(
-        choices=STATUS_CHOICES, max_length=50, default=Status.CONFIRMED
-    )
     reinvite = models.BooleanField(default=False)
     stopped_reasons = models.JSONField(null=True, blank=True)
+
+    @cached_property
```

Comment:
I didn't think to use @cached_property for the clinic status PR. If this approach makes sense, I'll apply the same change there in a follow up PR.